prov/efa/src/rdm/rxr_rma.c

```diff
@@ -219,6 +172,25 @@ ssize_t rxr_rma_readmsg(struct fid_ep *ep, const struct fi_msg_rma *msg, uint64_
 		err = -FI_EAGAIN;
 		goto out;
 	}
+
+	if (peer->is_local && rxr_ep->use_shm_for_tx) {
+		tmp_addr = msg->addr;
+		tmp_desc = msg->desc;
+		msg_clone = (struct fi_msg_rma *)msg;
```

Comment:
Is there a way in peer api to translate MR desc between peers, instead of 'cloning' and resetting msg?

Reply:
No such API exists, we can propose one, but in my experiment this desc convert does not impact performance